src/makeCalculator.js

```diff
@@ -36,8 +36,49 @@
  *
  * @return {object}
  */
+
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    operate(callback, value) {
+      calculator.result = callback(calculator.result, value);
+
+      return calculator;
```

Comment:
It is better to use `this` in such cases, to get right context here you can use `call`